indexer-standalone/src/main.rs

```diff
@@ -146,31 +147,39 @@ async fn run() -> anyhow::Result<()> {
         )
     });
 
+    info!("indexer shutting down");
```

Comment:
I know this is not directly related to this PR, but nevertheless it is obsolete, because the components already provide comprehensive logging wrt shutdown. Please remove this line.

Reply:
I left it as it was before with just diff on line 35